pex/bin/pex.py

```diff
@@ -290,8 +290,43 @@ def configure_clp_pex_environment(parser):
   group.add_option(
       '--platform',
       dest='platform',
-      default=Platform.current(),
-      help='The platform for which to build the PEX.  Default: %default')
+      default=None,
+      help='The platform for which to build the PEX. Defaults to the platform '
+           'of the running system.')
+
+  group.add_option(
+      '--python-version',
+      dest='python_version',
+      metavar='python_version',
+      default=None,
+      help=("Python version for which to to build the PEX. If not specified, "
+            "then the current system interpreter minor version is used. A major"
+            " version (e.g. '2') can be specified to match all "
+            "minor revs of that major version.  A minor version "
+            "(e.g. '34') can also be specified."),
+  )
+
+  group.add_option(
+      '--implementation',
+      dest='implementation',
+      metavar='implementation',
+      default=None,
+      help=("Python implementation for which to build the PEX (e.g. 'pp', 'jy',"
+            " 'cp',  or 'ip'). If not specified, then the current interpreter "
+            "implementation is used.  Use 'py' to force implementation-"
+            "agnostic wheels."),
```

Comment:
I went ahead and added `--tags`, and you can let me know if that solution seems good enough or not.